src/commands/init.rs

```diff
@@ -80,12 +80,19 @@ pub(crate) fn save_config(
     let key = Key::new();
 
     let pass = password.map_or_else(
-        || match prompt_password("enter password for new key: ") {
-            Ok(it) => it,
-            Err(err) => {
-                status_err!("{}", err);
-                RUSTIC_APP.shutdown(Shutdown::Crash);
-            }
+        || -> String {
+            let password_result = Password::new().with_prompt("enter password for new key")
+                .with_confirmation("confirm password", "passwords do not match")
```

Comment:
According to the documentation you have to enable empty passwords explicitly. These are currently supported by rustic, so please add `allow_empty_password(true)`.